src/powershell-native/nativemsh/pwrshcommon/CMakeLists.txt

```diff
@@ -10,6 +10,18 @@ add_library(pwrshcommon
 
 target_include_directories(pwrshcommon PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
+#
+# VS 2017 corrupts the INCLUDE environment variable when processing the csproj
+# file generated for this template. The netfxsdk path is shortened from
+# C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\Include\um
+# to
+# Include\um
+# This prevents msbuild from locating corerror.h and throws a resolution error
+# during compilation. We work around the problem by force-including the
+# contents of the environment variable before it gets corrupted.
+#
+target_include_directories(pwrshcommon PUBLIC $ENV{INCLUDE})
```

Comment:
OK. I checked out this PR and verified that it builds fine with VS 2015 installed only and the generated plugin binary works.